src/page/home/community/community.tsx

```diff
@@ -0,0 +1,114 @@
+import type { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Card } from '@/component/card/card'
+import { useMemo } from 'react'
+import { shallow } from 'zustand/shallow'
+import { Icon } from '@/component/icon/icon'
+import { useStatsAggregate } from '@/domain/stats/aggregate'
+import { useAppStore } from '@/store/appStore'
+import '../i18n/index'
+import './community.scss'
+
+type StatProps = {
+  description: string
+  stat: number
+  icon?: JSX.Element
+}
+
+type SingleStat = StatProps & {
+  _tag: 'singlestat'
+}
+
+type MultiStat = {
+  _tag: 'multistat'
+  stats: StatProps[]
+}
+
+type StatCard = SingleStat | MultiStat
+
+const Stat: FC<StatProps> = ({ description, stat, icon }): JSX.Element => (
+  <div className="okp4-dataverse-portal-home-page-stat-main">
+    {icon && <div className="okp4-dataverse-portal-home-page-stat-icon-container">{icon}</div>}
+    <div className="okp4-dataverse-portal-home-page-stat-content-wrapper">
+      <h2>{stat}</h2>
+      <p>{description}</p>
+    </div>
+  </div>
+)
+
+// eslint-disable-next-line max-lines-per-function
+export const Community: FC = () => {
+  const { t } = useTranslation('home')
+  const theme = useAppStore(store => store.theme)
+  const { dataspacesCreatedNb, datasetsSharedNb, servicesSharedNb, communityParticipantNb } =
+    useStatsAggregate(
+      aggregate => ({
+        dataspacesCreatedNb: aggregate.dataspacesCreatedNb,
+        datasetsSharedNb: aggregate.datasetsSharedNb,
+        servicesSharedNb: aggregate.servicesSharedNb,
+        communityParticipantNb: aggregate.communityParticipantNb
+      }),
+      shallow
+    )
+
+  const statCards: StatCard[] = useMemo(
```

Comment:
Great job, it's much better and readable. 👏
I'll just highlight a warning for the future, because, for me, this object is not extensible enough as the position is not explicit: you directly interpret it from the tag, but, if you have multiple cards within the same tag, how can you distinguish them? 😉

Reply:
Yes you're right